1_Lyalyushkin/ObjcBindings.py

```diff
@@ -0,0 +1,127 @@
+import ctypes.util
+from ctypes import c_char_p
+
+from ObjcConstants import CFStringRef, CFStringEncoding, CFIndex, CFAllocatorRef, CFTypeRef, \
+    CFDictionaryRef, \
+    CFErrorRef, kCFStringEncodingUTF8, CFArrayRef
+
+
+class CoreFoundation:
+    def __init__(self):
+        self.__core_foundation = ctypes.cdll.LoadLibrary(ctypes.util.find_library('CoreFoundation'))
+        self.__init_core_foundation()
+
+    def __init_core_foundation(self):
+        self.__core_foundation.CFStringGetCStringPtr.argtypes = [CFStringRef, CFStringEncoding]
+        self.__core_foundation.CFStringGetCStringPtr.restype = c_char_p
+
+        self.__core_foundation.CFStringGetCString.argtypes = [CFStringRef, c_char_p, CFIndex, CFStringEncoding]
+        self.__core_foundation.CFStringGetCString.restype = ctypes.c_bool
+
+        self.__core_foundation.CFStringCreateWithCString.argtypes = [CFAllocatorRef, c_char_p, CFStringEncoding]
+        self.__core_foundation.CFStringCreateWithCString.restype = CFStringRef
+
+        self.__core_foundation.CFArrayGetCount.argtypes = [CFArrayRef]
+        self.__core_foundation.CFArrayGetCount.restype = CFIndex
+
+        self.__core_foundation.CFArrayGetValueAtIndex.argtypes = [CFArrayRef, CFIndex]
+        self.__core_foundation.CFArrayGetValueAtIndex.restype = CFTypeRef
+
+        self.__core_foundation.CFDictionaryGetValue.restype = CFStringRef
+        self.__core_foundation.CFDictionaryGetValue.argtypes = [CFDictionaryRef, CFStringRef]
+
+        setattr(self.__core_foundation, 'kCFAllocatorDefault',
+                CFAllocatorRef.in_dll(self.__core_foundation, 'kCFAllocatorDefault'))
+        setattr(self.__core_foundation, 'CFIndex', CFIndex)
+        setattr(self.__core_foundation, 'CFStringRef', CFStringRef)
+        setattr(self.__core_foundation, 'CFTypeRef', CFTypeRef)
+        setattr(self.__core_foundation, 'CFAllocatorRef', CFAllocatorRef)
+        setattr(self.__core_foundation, 'CFArrayRef', CFArrayRef)
+        setattr(self.__core_foundation, 'CFDictionaryRef', CFDictionaryRef)
+        setattr(self.__core_foundation, 'CFErrorRef', CFErrorRef)
+        pass
+
+    def cf_array_get_count(self, cf_array_ref):
+        return self.__core_foundation.CFArrayGetCount(cf_array_ref)
+
+    def cf_dictionary_get_value(self, cf_dictionary_ref, key):
+        return self.__core_foundation.CFDictionaryGetValue(cf_dictionary_ref, key)
```

Comment:
How about creating `class CFDict` with overridden methods `__contains__`, `__len__`, `__getitem__`? It can help to eliminate long method names. Say `if cf_array_get_count(d) > 1` will be transformed to `if len(d) > 1` which is more pythonic.